### PR TITLE
Probably fixes ticket command not working on long tickets

### PR DESCRIPTION
**Pull request by adamsong**

### CODE REVIEW

File: src/main/java/net/yogstation/yogbot/listeners/commands/TicketCommand.kt [hunk 1]
```diff
@@ -103,20 +103,38 @@ class TicketCommand(
...
+
 		while (resultSet.next()) {
 			hasData = true
-			builder.append(resultSet.getTimestamp("when").toString())
-			builder.append(": ").append(resultSet.getString("user"))
-			builder.append(": ").append(resultSet.getString("text")).append("\n")
+			val newline = "${resultSet.getTimestamp("when")}: " +
+				"${resultSet.getString("user")}: " +
+				"${resultSet.getString("text")}\n"
+			if (builder.length + newline.length > 3990) {
```
Comment (AshCorr) [inline src/main/java/net/yogstation/yogbot/listeners/commands/TicketCommand.kt]: Assuming this is the discord message length limit? Do bot accounts get 4k characters, i was under the impression only nitro users got 4k characters and everyone else got 2k.

**Review by AshCorr — APPROVED** [review]

**Comment by AshCorr** [comment]
Looks good, just one comment on discord message lengths that I'm not sure is important.

File: src/main/java/net/yogstation/yogbot/listeners/commands/TicketCommand.kt (hunk 1 above)
Comment (AshCorr) [reply src/main/java/net/yogstation/yogbot/listeners/commands/TicketCommand.kt]: And maybe its worth defining it as a constant instead of a magic number

Comment (adamsong) [reply src/main/java/net/yogstation/yogbot/listeners/commands/TicketCommand.kt]: Just checked, the API docs say 2k, will fix

**Comment by AshCorr** [comment]
Build is failing

**Comment by adamsong** [comment]
Forgot to push the fix

**Pull request merged**